lib/pavilion/plugins/commands/clean.py

```diff
@@ -46,6 +47,53 @@ def _setup_arguments(self, parser):
     def run(self, pav_cfg, args):
         """Run this command."""
 
+        try:
+            self.validate_args(args)
+        except (commands.CommandError, ValueError) as err:
+            output.fprint("Command Argument Error:",
+                          color=output.RED, file=self.errfile)
+            output.fprint(err)
+            return errno.EINVAL
+
+        if args.verbose:
+            end = '\n'
+        else:
+            end = ''
```

Comment:
Not using a newline will make insanely ugly output. What are you trying to do here?

Reply:
And it looks like it only saves you three lines of output. 

Use a carriage return instead: `\r`

Four lines to say that is a bit silly too. 
```python
end = '\n' if args.verbose else '\r'
```

Reply:
The output in the delete function can similarly end in `\r` when not verbose, and use the `clear` option to fprint.